site/src/pages/docs/advanced-structured-logging.mdx

```diff
@@ -79,6 +79,94 @@ retrieving the information collected during request life cycle in a machine-frie
 | `responseContentPreview`              | the preview of the response content                                  |
 +---------------------------------------+----------------------------------------------------------------------+
 
+The duration of `totalDurationNanos` is different in the client-side and the server-side.
```

Comment:
How about this:

    The `totalDurationNanos` value is calculated differently between the client- and server-side.